devhub/devenv-readme.md

```diff
@@ -1,17 +1,19 @@
 # Relution Managed Development Environment
 
-This repository was set up by Relution as a managed development environment. This enabled developers registered in Relution to use it via SSH and create apps which will be uploaded to Relution.
+This repository was set up by Relution (https://www.relution.io) as a managed development environment. 
+It enables developers, which are member of a specific Relution account, to use it via SSH and create apps which will be uploaded to Relution.
```

Comment:
> which are member of a specific Relution account

Is this what you intended to write? Shouldn't this be _owner_ or just _which own/have..